tasks.py.jinja

```diff
@@ -283,14 +286,20 @@ def lint(c, verbose=False):
 
 
 @task(develop)
-def start(c, detach=True, debugpy=False):
+def start(c, detach=True, debugpy=False, wait=None):
     """Start environment."""
     cmd = "docker-compose up"
     if detach:
         cmd += " --detach"
     with c.cd(str(PROJECT_ROOT)):
         c.run(cmd, env=dict(UID_ENV, DOODBA_DEBUGPY_ENABLE=str(int(debugpy))))
-
+    if wait:
+        _logger.info("Waiting for services to spin up...")
+        debugpy_log = None
+        while(not debugpy_log or b"Adapter is accepting incoming client connections" not in debugpy_log):
+            cmd = "docker-compose exec -T odoo cat /tmp/debugpy-log/debugpy.server-1.log"
+            debugpy_log = subprocess.run(cmd.split(' '), stdout=subprocess.PIPE, stderr=subprocess.PIPE).stdout
```

Comment:
Use `c` instead of `subprocess`, to be consistent. Invoke provides it, with a handful of niceties.